bindings/pyroot/pythonizations/python/ROOT/_pythonization/_function_utils.py

```diff
@@ -0,0 +1,230 @@
+from .._numbadeclare import _NumbaDeclareDecorator
+
+try:
+    import re
+except:
+    raise ImportError("Failed to import re during call to determine function signature.")
+
+try:
+    import numpy as np
```

Comment:
This numpy import is still not lazy, since `_function_utils` is imported at the top of `_rdataframe`, this means it will be always attempted at `import ROOT` time, when the pythonizors for all classes are registered. To make it lazy, you need to include it when it is used (in this case, the `find_type` method).